web/src/click_handlers.js

```diff
@@ -565,6 +567,10 @@ export function initialize() {
             );
         }
 
+        if (user_status.get_status_text(user_id)) {
+            return;
```

Comment:
These early returns are non-starters--if a user has a status set, the regular buddy-list tooltip will not be rendered. And that's a significant regression.